src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -207,6 +207,13 @@ public void onCreate(Bundle paramBundle) {
       }
       return true;
     }));
+
+    if (dcContext.isChatmail()) {
+      sentboxWatchCheckbox.setVisible(false);
+      bccSelfCheckbox.setVisible(false);
```

Comment:
if someone that is already using chatmail disabled this toggle, now they will not be able to enable it anymore, would be nice to do some migration checking for this property and maybe also the other ones, and set/force the correct default values

Reply:
imu, whatever is chosen for these four options, it does not make a difference for chatmail. but not sure - otherwise, i would prefer some migration in the core, or that the settings are ignored, at least when they're harmful, i think, we do want to have less footguns for chatmail :) cc @link2xt

Reply:
bcc self makes a difference without it multidevice sync is broken, or do you mean that internally core already ignores their value if chatmail flag is set?